test/NameRegistry.t.sol

```diff
@@ -235,6 +235,51 @@ contract NameRegistryTest is Test {
         nameRegistry.makeCommit(commitHash);
     }
 
+    /*//////////////////////////////////////////////////////////////
+                            VIEW FUNCTIONS
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice Get the recovery address of a fname.
+     *
+     * @param tokenId The uint256 representation of the fname
+     * @return recovery The address which can recover the fname
+     */
+
+    function _recovery(uint256 tokenId) internal view returns (address recovery) {
+        (recovery,) = nameRegistry.registrationMetadataOf(tokenId);
+    }
+
+    /**
+     * @notice Get the expiration time of a fname.
+     *
+     * @param tokenId The uint256 representation of the fname
+     * @return expiryTs The timestamp when the fname expires
+     */
+    function expiry(uint256 tokenId) internal view returns (uint256 expiryTs) {
+        (, expiryTs) = nameRegistry.registrationMetadataOf(tokenId);
+    }
+
+    /**
+     * @notice Get the recovery destination of a fname.
+     *
+     * @param tokenId The uint256 representation of the fname
+     * @return _recoveryDestination The destination address of the most recent recovery attempt.
+     */
+    function recoveryDestination(uint256 tokenId) internal view returns (address _recoveryDestination) {
+        (_recoveryDestination,) = nameRegistry.recoveryMetadataOf(tokenId);
+    }
+
+    /**
+     * @notice Get the recovery clock of a fname.
+     *
+     * @param tokenId The uint256 representation of the fname
+     * @return recoveryTs The timestamp of the recovery attempt or zero if there is no active recovery.
+     */
+    function _recoveryTs(uint256 tokenId) internal view returns (uint256 recoveryTs) {
```

Comment:
self: use consistent naming for the view functions, give them all underscores or a prefix